Makefile

```diff
@@ -204,7 +204,7 @@ build/codegenv2:
 	  --go-grpc_out=require_unimplemented_servers=false,paths=source_relative:pkg/api/protomodel \
 	  --grpc-gateway_out=logtostderr=true,paths=source_relative:pkg/api/protomodel \
 	  --doc_out=pkg/api/protomodel --doc_opt=markdown,docs.md \
-	  --openapiv2_out=allow_merge=true,simple_operation_ids=true,disable_service_tags=true:pkg/api/openapi \
+	#   --openapiv2_out=allow_merge=true,simple_operation_ids=true,disable_service_tags=true:pkg/api/openapi \
```

Comment:
spec needs to be regenerated, note the generation plugin has recently changed in base branch

Reply:
fails for the swagger plugin